crates/net/tests/http.rs

```diff
@@ -3,8 +3,6 @@ use once_cell::sync::Lazy;
 use serde::{Deserialize, Serialize};
 use wasm_bindgen_test::*;
 
-wasm_bindgen_test_configure!(run_in_browser);
```

Comment:
This is needed for browser tests, please keep this in (and gate it behind `wasm32-unknown-unknown` target)

Reply:
Clippy doesn't like `#[cfg(target="wasm32-unkonwn-unknown")]` but it seems to work. Let me know if I misunderstood.

Reply:
I think `target_arch` refers to wasm32 in this case (see https://doc.rust-lang.org/reference/conditional-compilation.html#target_arch). That might be why clippy complains. You want to update CI so clippy is also run on wasm32 target.

You can look at how wasi is cfg-gated in other parts of the code base and do the opposite of it here. If it's `target_os = "wasi"` then you may be able to do `not(target_os = "wasi")` to target browser wasm targets

Reply:
So this seems like a quirk of `wasm_bindgen_test`. As far as I can tell there's no way straightforward way to run in both node and browser.

Here's one suggestion: https://github.com/rustwasm/wasm-bindgen/issues/2571#issuecomment-873533119

Added to latest commit.